### ISSUE TITLE
vmagent cannot scrape mTls targets  - bad certificate

### ISSUE THREAD

**Issue by f41gh7** [VictoriaMetrics/VictoriaMetrics#470]
**Describe the bug**

Cannot scrape metrics from mTls targets.
error:
```
 VictoriaMetrics/lib/promscrape/scrapework.go:175        error when scraping "https://172.xxx.xxx.xxx:2379/metrics" from job "kubernetes-etcd" with labels {instance="172.xxx.xxx.xxx:2379", job="kubernetes-etcd"}: error when scraping "https://172.xxx.xxx.xxx:2379/metrics": remote error: tls: bad certificate

```
**To Reproduce**

Inside k8s configuration for scape
```yaml
    - job_name: 'kubernetes-etcd'
      kubernetes_sd_configs:
        - role: endpoints
      scheme: https
      tls_config:
        ca_file: /etc/secrets/etcd_issuingCa
        cert_file: /etc/secrets/etcd_certificate
        key_file: /etc/secrets/etcd_privateKey
        insecure_skip_verify: true
      relabel_configs:
          - source_labels: [__meta_kubernetes_namespace, __meta_kubernetes_service_name, __meta_kubernetes_endpoint_port_name]
            action: keep
            regex: default;kubernetes;https
          - source_labels: [__address__]
            regex: '(.*):.*'
            replacement: '${1}:2379'
            target_label: __address__
```
**Expected behavior**

**Screenshots**

**Version**
The line returned when passing `--version` command line flag to binary. For example:
```
./vmagent-prod --version
vmagent-20200507-201403-tags-v1.35.3-0-gefc8e3c5
```

**Additional context**
curl works, prometheus scrape was ok as well.
```
curl https://172.xxx.xxx.xxx:2379/metrics --cacert etcd_issuingCa --key etcd_privateKey --cert etcd_certificate
# HELP etcd_debugging_mvcc_db_compaction_pause_duration_milliseconds Bucketed histogram of db compaction pause duration.
# TYPE etcd_debugging_mvcc_db_compaction_pause_duration_milliseconds histogram
```

**Comment by valyala** [comment]
@f41gh7 , could you verify whether `vmagent` v1.35.0 works as expected?

**Comment by f41gh7** [comment]
yep, it`s ok 
```curl localhost:8429/targets
job="kubernetes-etcd" (3/3 up)
```

**Comment by valyala** [comment]
The bug should be fixed in the commit 0a134ace63d78a9b2bd94409708a7db329b4a876 . This commit will be included in the next release.

@f41gh7 , could you build `vmagent` from this commit according [to these docs](https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/app/vmagent/README.md#how-to-build-from-sources) and verify whether the big is fixed there?

**Comment by f41gh7** [comment]
it`s ok now, thx!

**Comment by valyala** [comment]
The bugfix for proper scraping of mTLS targets has been included in [v1.35.4](https://github.com/VictoriaMetrics/VictoriaMetrics/releases).
